kde-frameworks/kdelibs4support/kdelibs4support-5.109.0.ebuild

```diff
@@ -86,17 +87,13 @@ PATCHES=(
 
 src_prepare() {
 	ecm_src_prepare
-
-	if ! use handbook; then
-		sed -e "/kdoctools_install/ s/^/#DONT/" -i CMakeLists.txt || die
-	fi
-
```

Comment:
Okay, I've had to revisit this handbook mechanism on my own to remember why we had this here.

1. Per our discussion back in https://github.com/gentoo/gentoo/pull/32474#pullrequestreview-1597184693 we established that `ECM_HANDBOOK="forceoptional"` is not possible for this package.
2. Due to `ECM_HANDBOOK="true"`, the `kdoctools_install` call is not being dropped by `ecm.eclass`
3. So wenn we build with `USE=-handbook`, you made sure that `ECM_HANDBOOK_DIR="docs"` subdir would be disabled, *but* translated doc residing within `po/L10N/doc` would still be installed

kdelibs4support is just that bit special. We'll have to keep that manual `sed` here.